modules/importing-manifest-list-through-imagestreamimport.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+// * assembly/openshift_images/managing-image-streams.adoc
+
+:_content-type: PROCEDURE
+[id="importing-manifest-list-through-imagestreamimport_{context}"]
+= Importing a manifest list through ImageStreamImport
+
+Use the following procedure to import a manifest list through the `ImageStreamImport` object with the `importMode` value. 
+
+.Procedure
+
+. Create an `ImageStreamImport` object that sets the `importMode` to `Manifestlist` by running the following command:
++
+[source,terminal]
+----
+$ cat <<EOF | oc create -f -
+apiVersion: image.openshift.io/v1
+kind: ImageStreamImport
+metadata:
+  name: app
+  namespace: <namespace>
+spec:
+  import: true
+  images:
+  - from:
+      kind: DockerImage
+      name: quay.io/<user_name>/<image_name>
+    to:
+      name: latest
+    referencePolicy:
+      type: Source
+    importPolicy:
+      importMode: "ManifestList"
+EOF
+----
+
+. Enter the following command to ensure that Image objects were created for the manifest list: 
++
+[source,terminal]
+----
+$ oc get image | grep quay.io/<user_name>/<image_name>
+----
+
+. Enter the following command to ensure that all sub-manifests were created:
```

Comment:
the provided command doesn't tell you about sub-manifests, and most users don't care how it is imported; the thing they care about - the image reference from `from.name` will be resolved into an image reference with sha and it'll work on all architectures that are supported by the source image.

Reply:
So I can remove 

. Enter the following command to ensure that all sub-manifests were created:
+
[source,terminal]
----
$ oc get is app
----

?